dashboard/modules/job/cli.py

```diff
@@ -136,7 +136,7 @@ def submit(
     entrypoint: Tuple[str],
     no_wait: bool,
 ):
-    """Submits a job to be run on the cluster docstring
```

Comment:
lol wat

Reply:
debris from earlier sphinx debugging that I forgot to clean up :(